lexers/h/html.go

```diff
@@ -56,4 +57,11 @@ var HTML = internal.Register(MustNewLexer(
 			{`[^\s>]+`, LiteralString, Pop(1)},
 		},
 	},
-))
+).SetAnalyser(func(text string) float32 {
+	matched, _ := doctype.MatchString(text, "html")
+	if matched {
```

Comment:
You can do this in one line
```suggestion
	if matched, _ := doctype.MatchString(text, "html"); matched {
```